torch/utils/_benchmark/utils/valgrind_wrapper/timer_interface.py

```diff
@@ -0,0 +1,448 @@
+                     Source information may be limited. Rebuild with
+                     REL_WITH_DEB_INFO=1 for more detailed results.""")
+        return output
+
+    def stats(self, inclusive: bool = False) -> Tuple[Tuple[int, str], ...]:
+        """Returns stats as a tuple of (count, function)
+
+        `inclusive` matches the semantics of callgrind. If True, the counts
+        include instructions executed by children. `inclusive=True` is useful
+        for identifying hot spots in code; `inclusive=False` is useful for
+        identifying reducing noise when diffing counts from two different
+        runs. (See CallgrindStats.delta(...) for more details)
+        """
+        if inclusive:
+            first, second = self.stmt_inclusive_stats, self.baseline_inclusive_stats
+        else:
+            first, second = self.stmt_exclusive_stats, self.baseline_exclusive_stats
+        return self._diff(first, second)
+
+    def counts(self, include_lookdict_unicode: bool = True) -> int:
+        """Returns the total number of instructions executed.
+
+        Several instructions in the CPython interpreter are rather noisy. These
+        instructions involve unicode to dictionary lookups which Python uses to
+        map variable names. By default these are included, but setting
+        `include_lookdict_unicode=False` will exclude them and generally lead
+        to less noisy counts.
+        """
+        return self._counts(self.stmt_exclusive_stats, include_lookdict_unicode)
+
+    # FIXME: Once 3.7 is the minimum version, type annotate `other` per PEP 563
+    def delta(self, other, inclusive: bool = False, subtract_baselines: bool = True):
+        """Diff two sets of counts.
+
+        One common reason to collect instruction counts is to determine the
+        the effect that a particular change will have on the number of instructions
+        needed to perform some unit of work. If a change increases that number, the
+        next logical question is "why". This generally involves looking at what part
+        if the code increased in instruction count. This function automates that
+        process so that one can easily diff counts on both an inclusive and
+        exclusive basis. The `subtract_baselines` argument allows one to disable
+        baseline correction, though in most cases it shouldn't matter as the
+        baselines are expected to more or less cancel out.
+        """
+        if subtract_baselines:
+            first = self.stats(inclusive=inclusive)
+            second = other.stats(inclusive=inclusive)
+        else:
+            if inclusive:
+                first, second = self.stmt_inclusive_stats, other.stmt_inclusive_stats
+            else:
+                first, second = self.stmt_exclusive_stats, other.stmt_exclusive_stats
+        return self._diff(first, second)
+
+    # FIXME: Once 3.7 is the minimum version, type annotate output
+    def as_standardized(self):
```

Comment:
You don't need to wait, just wrap the output in quotes: `as_standardized(self) -> 'CallgrindStats'`

Reply:
Ohh. Nice.

Reply:
Done.